source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -17,11 +17,17 @@
 import omni.isaac.lab.utils.math as math_utils
 from omni.isaac.lab.assets import Articulation, RigidObject
 from omni.isaac.lab.managers import SceneEntityCfg
+from omni.isaac.lab.managers.manager_base import ManagerTermBase
+from omni.isaac.lab.managers.manager_term_cfg import ObservationTermCfg
 from omni.isaac.lab.sensors import Camera, RayCaster, RayCasterCamera, TiledCamera
 
 if TYPE_CHECKING:
     from omni.isaac.lab.envs import ManagerBasedEnv, ManagerBasedRLEnv
 
+from transformers import AutoModel
+
+from torchvision import models
```

Comment:
How about moving these imports to inside the observation term class where they are used. Since these are optional dependencies, we don't need to import them at global level of the module.